enterprise/backend/test/metabase_enterprise/advanced_permissions/api/channel_test.clj

```diff
@@ -36,8 +36,7 @@
               (update-channel user 403)
               (create-channel :crowberto 200)
               (update-channel :crowberto 200)
-              (include-details :crowberto true)
-              (include-details user false)))
```

Comment:
should not we still check that some user doesn't have access to webhooks?

Reply:
good point, added in [9301055](https://github.com/metabase/metabase/pull/53210/commits/9301055fc02f1ae8f914d062269c9db7e46bf9df)